contrib/kind.sh

```diff
@@ -171,7 +171,7 @@ OVN_IMAGE=${OVN_IMAGE:-local}
 # NOTE: Upstream KIND IPv6 masks are different (currently rejected by ParseClusterSubnetEntries()):
 #  Upstream - NET_CIDR_IPV6=fd00:10:244::/64 SVC_CIDR_IPV6=fd00:10:96::/112
 NET_CIDR_IPV4=${NET_CIDR_IPV4:-10.244.0.0/16}
-SVC_CIDR_IPV4=${SVC_CIDR_IPV4:-10.96.0.0/12}
+SVC_CIDR_IPV4=${SVC_CIDR_IPV4:-10.96.0.0/16}
```

Comment:
are there compatibility implications for changing this default? will this attempt to change an existing cluster?

Reply:
this is just for cluster creation,  does not modify any existing cluster.
did you mind to review this one Jordan? :smile: